sections/semantics-forms.include

```diff
@@ -7348,7 +7348,7 @@ You cannot submit this form when the field is incorrect.</samp></pre>
     </dd>
   </dl>
 
-  The <{button}> element <a>represents</a> a button labeled by its contents.
+  The <{button}> element <a>represents</a> an interactive control, labeled by its contents, that allows for, if enabled,  user-triggered actions when activated.
 
```

Comment:
I second this. Button is a well established concept (well, aside from coat buttons, thanks English) and I get the impression that this makes the definition less clear.